src/screens/Exchange/Exchange.js

```diff
@@ -831,6 +864,7 @@ class ExchangeScreen extends React.Component<Props, State> {
         >
           {!isSubmitted && <HotSwapsHorizontalList onPress={this.onSwapPress} swaps={swaps} />}
           <FormWrapper bottomPadding={isSubmitted ? 6 : 30}>
+            {this.shouldShowSellMax() && <SellMaxButton onPress={this.handleSellMax} />}
```

Comment:
What do you think about adding this as a TextInput element?
Would be more reusable and cleaner (no absolute positioning needed).
As for this component itself, we also have ButtonText component that has native Android feel.
Maybe it could be reused here :)